addons/web_editor/static/lib/odoo-editor/src/utils/utils.js

```diff
@@ -1389,6 +1389,15 @@ export function getOuid(node, optimize = false) {
     }
     return node && node.oid;
 }
+/**
+ * Returns true if the provided node can suport html content
```

Comment:
Since you'll be returning to this PR because of the other comment anyway, could you please put a dot at the end of the sentence here ? Thank you ! It's the last wave of review comments on this PR, after that I'll r+ it ! :p